apps/summary.py

```diff
@@ -567,8 +580,41 @@ def store_query(name):
         pdftracklet = pd.read_json(r.content)
     else:
         pdftracklet = pd.DataFrame()
+
     return pdfs.to_json(), pdfsU.to_json(), pdfsUV.to_json(), pdfsso.to_json(), pdftracklet.to_json()
 
+@app.callback(
+    Output('object-release', 'children'),
+    Input('lightcurve_request_release', 'n_clicks'),
+    State('object-data', 'children'),
+    prevent_initial_call=True,
+    background=True,
+    running=[
+        (Output('lightcurve_request_release', 'disabled'), True, False),
+        (Output('lightcurve_request_release', 'loading'), True, False),
+        # (Output('lightcurve_request_release', 'children'), 'Requesting data release photometry...', 'Data release photometry requested'),
+    ],
+)
+def store_release_photometry(n_clicks, object_data):
+    if not n_clicks or not object_data:
+        return no_update
+
+    pdf = pd.read_json(object_data)
+
+    mean_ra = np.mean(pdf['i:ra'])
+    mean_dec = np.mean(pdf['i:dec'])
+
+    pdf_release = pd.read_csv(
+        'https://irsa.ipac.caltech.edu/cgi-bin/ZTF/nph_light_curves?POS=CIRCLE%20{}%20{}%20{}&BAD_CATFLAGS_MASK=32768&FORMAT=CSV'.format(
+            mean_ra,
+            mean_dec,
+            2/3600
+        )
+    )
+    pdf_release = pdf_release[['mjd', 'mag', 'magerr', 'filtercode']]
```

Comment:
What if there is no match, i.e. `pdf_release` is empty? We could catch the exception, and later display a message about the status of the query?

Reply:
Added some feedback for successful / unsuccessful loading of the data.